mastering-plone/relations.rst

```diff
@@ -68,15 +68,15 @@ For even more flexibility you can create your own `dynamic vocabularies <https:/
 Accessing and displaying related items
 --------------------------------------
 
-One would think that it would be the easiest approach to simply use the render-method of the default-widget like we did in the chapter "Views II: A Default View for “Talk”". Sadly that is wrong. Adding the approriate code to te template:
+One would think that it would be the easiest approach to simply use the render-method of the default-widget like we did in the chapter "Views II: A Default View for “Talk”". Sadly that is wrong. Adding the appropriate code to the template:
 
-..  code-block::html
+..  code-block:: html
```

Comment:
Remove one more extra space.

```
.. code-block:: html
```

Reply:
This space after the colon is the fix. Without the space it don't work as expected. See: #388

Reply:
You should remove the extra space because it gets interpreted as a comment with the extra space. See http://docutils.sourceforge.net/docs/ref/rst/directives.html 

> Directives have the following syntax:
> 
>     +-------+-------------------------------+
>     | ".. " | directive type "::" directive |
>     +-------+ block                         |
>             |                               |
>             +-------------------------------+
> Directives begin with an explicit markup start (two periods and **a space**), followed by the directive type and two colons (collectively, the "directive marker").

Not **two spaces**.

Reply:
Got it, I didn't realized you were talking about the spaces between the markup and "code-block".

Reply:
Done.